.circleci/config.yml

```diff
@@ -94,6 +94,13 @@ jobs:
             find . -type f -regex  ".*/test-results.*\.xml" -exec cp {} $CIRCLE_TEST_REPORTS/ \;
           when: always
 
+      - run:
+          name: Build Notifications Panel
+          command: |
+            if [ "$CIRCLE_NODE_INDEX" == "0" ]; then
+                NODE_ENV=production npm run sdk -- notifications --output-dir=$CIRCLE_ARTIFACTS/notifications-panel
```

Comment:
Folks should be able to get related artifact off of `https://circleci.com/api/v1.1/project/github/Automattic/wp-calypso/latest/artifacts?&branch=notifications/update-sdk-script&filter=successful`

Should we add a helper script like we have for string artifacts in https://github.com/Automattic/wp-calypso/blob/master/bin/get-circle-string-artifact-url

Highly recommended to also add an integration test here.

Reply:
Oh cool, this pairs with D18484-code

Reply:
updated D18484-code so that the process is almost fully automated